content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,440 @@
+                    "type": "host-groups"
+                }
+            ],
+            "associated_systems": [
+                "2M29510B8N",
+                "2M29510B8L"
+            ],
+            "comment": null,
+            "console_uri": "/data-ops-manager/host-initiators/e1b09ecf837b46cca365044bf7237bc3",
+            "contact": null,
+            "customer_id": "eb00678a466b11ec94d66ec0ab988305",
+            "edit_status": "Not_Applicable",
+            "fqdn": null,
+            "generation": 1653561376,
+            "host_groups": [
+                {
+                    "id": "0fb8a5f6616c4465a77a43cb7841e105",
+                    "marked_for_delete": false,
+                    "name": "HostGroup-01",
+                    "systems": [
+                        "2M29510B8N",
+                        "2M29510B8L"
+                    ],
+                    "user_created": true
+                }
+            ],
+            "id": "e1b09ecf837b46cca365044bf7237bc3",
+            "initiators": [
+                {
+                    "address": "c3:33:ff:58:5f:19:00:04",
+                    "id": "1b6b97741b21480bb916ca3b8c0a2c6e",
+                    "ip_address": null,
+                    "name": "Host Path C333FF585F190004 (0:3:1)",
+                    "protocol": "FC",
+                    "systems": [
+                        "2M29510B8N",
+                        "2M29510B8L"
+                    ]
+                },
+                {
+                    "address": "c3:33:ff:58:5f:19:00:24",
+                    "id": "1bce3ffd6a0d4b74a8520ddc55e2a1eb",
+                    "ip_address": null,
+                    "name": "Host Path C333FF585F190024 (1:3:1)",
+                    "protocol": "FC",
+                    "systems": [
+                        "2M29510B8N",
+                        "2M29510B8L"
+                    ]
+                }
+            ],
+            "ip_address": null,
+            "location": null,
+            "marked_for_delete": false,
+            "model": null,
+            "name": "Host-01",
+            "operating_system": "Ubuntu",
+            "persona": null,
+            "protocol": null,
+            "subnet": null,
+            "systems": [
+                "2M29510B8N",
+                "2M29510B8L"
+            ],
+            "type": "host-initiator",
+            "user_created": true
+        }
+    ]
+}
+
+PLAY RECAP ********************************************************************************************************************************************************************************************************
+127.0.0.1                  : ok=3    changed=0    unreacha-ble=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+In the current release, only critical resources are supported. New resources will be added moving forward. This is just the beginning.  In the future, there are many use cases to cover.
```

Comment:
It would be nice if you added a quick summary after this. Maybe something like this:
In this post, I introduced the Ansible playbooks for HPE GreenLake Data Services Cloud Console and showed you how to use them. Please check the HPE Developer blog often to see more posts related to the Data Services Cloud Console.